app/controllers/RasResidencyCheckerController.scala

```diff
@@ -50,11 +50,11 @@ trait RasResidencyCheckerController extends RasController {
           None
```

Comment:
equivalent to rasResponse.nextYearForecastResidencyStatus.map(extractResidencyStatus)

Reply:
Added replacement

Reply:
Please re-review @daniel-manning